docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -1406,14 +1399,13 @@ _Returns_
 
 <a name="showInsertionPoint" href="#showInsertionPoint">#</a> **showInsertionPoint**
 
-Sets the insertion point and shows it to users.
-
-Components like <Inserter> will default to inserting blocks at this point.
+Returns an action object used in signalling that the insertion point should
```

Comment:
Please note that I did a partial revert of 7f1fac93ffcfc8842b651b4448d301fb61caddcf.